pkgs/development/haskell-modules/configuration-freebsd.nix

```diff
@@ -0,0 +1,13 @@
+# FREEBSD-SPECIFIC OVERRIDES FOR THE HASKELL PACKAGE SET IN NIXPKGS
+
+{ pkgs, haskellLib }:
+
+let
+  inherit (pkgs) lib darwin;
+in
+
+with haskellLib;
+
+self: super: ({
+  time-compat = dontCheck super.time-compat;
```

Comment:
Could you leave a comment here why this needs `dontCheck`?  Ideally also link to an issue filed on the upstream repo.